### PR TITLE
Readded Keep testing files

### PR DESCRIPTION
**Pull request by daledesilva**
This pull request only adds an unaltered Google Keep export file for future testing and 2 jsons that don't match the google keep data structure.

### CODE REVIEW

**Comment by daledesilva** [comment]
Closed the previous PR related to this and created a new one because I wasn't struggling with some merge issues.

File: tests/keep/broken-json-note.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "color": "DEFAULT",
+    "isTrashed": false,
+    "isPinned": false,
+    "isArchived": false,
+    "textContent": "This is just a plain text note.     This sentence has more than one space before it.\n\n\nThis sentence has more than one new line before it.\n\n   This sentence has more than one new line and spaces before it.",
+    "title": "_Text note"
```
Comment (lishid) [inline tests/keep/broken-json-note.json:7]: I don't think we need to test whether the JSON parser can detect broken files. I think the actual exported zip file is good enough.

Comment (daledesilva) [reply tests/keep/broken-json-note.json:7]: You wouldn't wanna know if it can break with unexpected input?

Comment (lishid) [reply tests/keep/broken-json-note.json:7]: I don't think we need to test the browser implementation of JSON.parse

Comment (daledesilva) [reply tests/keep/broken-json-note.json:7]: Not for that, but to Test that the keep format-importer code is correctly try/catching and reporting.
(For the broken file)

And the other file tests that it gracefully handles and reports a valid json that isn't an valid keep json.

In both cases it should show that it fails in the output log.

Comment (lishid) [reply tests/keep/broken-json-note.json:7]: Oh ok, fair enough.

**Pull request merged**